test/server-custom-livereload.js

```diff
@@ -0,0 +1,120 @@
+import request from 'supertest';
+import assert from 'assert';
+import listen from './helpers/listen';
+import {PassThrough} from 'stream';
+
+describe('tiny-lr', () => {
+  before(listen({
+    livereload: function () {
+      const s = new PassThrough();
+
+      s.end('// custom live-reload');
+
+      return s;
+    }
+  }));
+
```

Comment:
All the below tests, except maybe one `GET /livereload.js`, are copy-pasted from `server.js`. Can you try to refactor these two files to use a function passed to the parent describe in order to not duplicate these tests ?

Something like:

```js

describe('tiny-lr custom livereload', () => {
  before(...)

  testLR(this);

  // divergent tests here, mainly GET /livereload.js
});
```

with testLR including all common tests from these two files. and using a param like `context` to map the `this` passed as a parameter.